secretsauce/apps/account/api.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth.models import User
+from django.db import transaction
+from rest_framework import generics, permissions
+from rest_framework.response import Response
+from knox.models import AuthToken
+
+from secretsauce.apps.account.models import MyUser
+from secretsauce.apps.account.serializer import RegisterSerializer, UserSerializer, LoginSerializer, MyUserSerializer
+
+
+# Register API
+class RegisterAPI(generics.CreateAPIView):
+  serializer_class = MyUserSerializer
+  permission_classes = (permissions.AllowAny,)
+  queryset = MyUser.objects.all()
+
+  @transaction.atomic
+  def post(self, request, *args, **kwargs):
+    serializer = MyUserSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+    user = serializer.save()
+    authuser = User.objects.create_user(username=request.data['username'], email=request.data['email'])
```

Comment:
After registering a user, the login functionality does not work.

I went to further dig and found on django admin that the password column is missing.
![image](https://user-images.githubusercontent.com/39354695/82336445-2ee96c00-9a1d-11ea-9203-fe6e2e11f7f7.png)

I believe what you are missing is this: 
```suggestion
    authuser = User.objects.create_user(username=request.data['username'], email=request.data['email'], password=request.data["password"])
```